### ISSUE TITLE
build fails with latest deno

### ISSUE THREAD

**Issue by derari** [bigskysoftware/missing#39]
Running with the latest version

```
docker run --rm -ti -v .:/app denoland/deno:alpine sh
# cd /app
# deno task css
Task css deno run -qA tool/css.ts src/missing.css && deno run -qA tool/css.ts src/missing-prism.css
error: Top-level await promise never resolved
const compile = await bundleAsync({
                ^
    at <anonymous> (file:///app/tool/css.ts:15:17)
```

for comparison, using an older version

```
docker run --rm -ti -v .:/app denoland/deno:alpine-1.35.3 sh
# cd /app
# deno task css
Task css deno run -qA tool/css.ts src/missing.css && deno run -qA tool/css.ts src/missing-prism.css
#
```

**Comment by dz4k** [comment]
Looks like a Deno-side bug unfortunately: https://github.com/denoland/deno/issues/20814

**Comment by dz4k** [comment]
this should be fixed now -- builds in `deno 1.42.3 (release, x86_64-unknown-linux-gnu)` for me. Could you test?
